### PR TITLE
[quantization] lower lmhlo_disc::quantized_dynamic_conv to library call

### PR DESCRIPTION
**Pull request by wyzero**
to #567

### CODE REVIEW

File: tao_compiler/mlir/disc/transforms/disc_lower_to_library_call.cc [hunk 1]
```diff
@@ -364,15 +369,19 @@ struct ConvConverter : public OpRewritePattern<ConvolutionOp> {
   }
 };
 
-struct DynamicConvConverter : public OpRewritePattern<DynamicConvOp> {
-  using OpRewritePattern<DynamicConvOp>::OpRewritePattern;
+template <typename OpTy>
+struct DynamicConvLikeConverter : public OpRewritePattern<OpTy> {
+  DynamicConvLikeConverter(MLIRContext* context, StringRef target)
+      : OpRewritePattern<OpTy>::OpRewritePattern(context) {
+    this->target_ = target;
```
[the remark this answers is not in the document]
Comment (qiuxiafei) [reply tao_compiler/mlir/disc/transforms/disc_lower_to_library_call.cc:376]: well ... it seems always valid since we always give a static string literal.

**Review by qiuxiafei — APPROVED** [review]
Generally LGTM with trivial comments.

File: tao_compiler/mlir/disc/IR/lhlo_disc_ops.cc [hunk 2]
```diff
@@ -39,8 +39,8 @@ LmhloDiscDialect::LmhloDiscDialect(MLIRContext* context)
 
 LogicalResult CustomCallOp::verify() {
   CustomCallOp op = *this;
-  if (op.target_arg_mapping()) {
-    lmhlo::CustomCallTargetArgMappingAttr mapping = *op.target_arg_mapping();
+  if (op.getTargetArgMapping()) {
```
[the remark this answers is not in the document]
Comment (wyzero) [reply tao_compiler/mlir/disc/IR/lhlo_disc_ops.cc:42]: Yes.

[the remark this answers is not in the document]
Comment (wyzero) [reply tao_compiler/mlir/disc/IR/lhlo_disc_ops.cc:42]: I add a line `let emitAccessorPrefix = kEmitAccessorPrefix_Prefixed;` in lmhlo_disc dialect definition to match the behaviour of lmhlo dialect.

**Pull request merged**